_DeploymentAndDistroPackaging/ansible/roles/ciao-controller/defaults/main.yml

```diff
@@ -77,3 +77,24 @@ ceph_id: admin
 
 # Path to ceph user keyring
 secret_path: /etc/ceph/ceph.client.admin.keyring
+
+# URL for the latest fedora cloud image
+fedora_cloud_image_url: https://download.fedoraproject.org/pub/fedora/linux/releases/24/CloudImages/x86_64/images/Fedora-Cloud-Base-24-1.2.x86_64.qcow2
+
+# Fedora cloud image file name
+fedora_cloud_image: "{{ fedora_cloud_image_url.split('/') | last }}"
+
+# URL for the latest ciao networking image
+cnci_image_url: https://download.clearlinux.org/demos/ciao/clear-8260-ciao-networking.img.xz
+
+# Ciao networking image file name
+cnci_image: "{{ cnci_image_url[0:-3].split('/') | last }}"
+
+# URL for the latest clearlinux cloud image
+clear_cloud_image_url: https://download.clearlinux.org/releases/11960/clear/clear-11960-cloud.img.xz
```

Comment:
That link will not get broken, the question would be, how often we'll be updating this URL?